core/amber/src/main/protobuf/edu/uci/ics/amber/engine/architecture/worker/controlreturns.proto

```diff
@@ -35,10 +35,9 @@ message ControlReturnV2 {
   oneof value {
     ControlException control_exception = 1;
 
-    WorkerStatistics worker_statistics = 2;
-    WorkerState worker_state = 3;
-    CurrentInputTupleInfo current_input_tuple_info = 4;
-    EvaluatedValue evaluated_value = 5;
+    WorkerInfo worker_info = 2;
```

Comment:
Agreed that we can use a new wrapper for this level of information and thanks for separating `WorkerState` from `WorkerStatstics`. I am concerned about the naming. We use `info` for other information. Maybe we can name this wrapper as `WorkerMatrics`?

Reply:
Fixed!